plugin/sampling/strategystore/static/strategy_store.go

```diff
@@ -59,7 +59,7 @@ func loadStrategies(strategiesFile string) (*strategies, error) {
 	if strategiesFile == "" {
 		return nil, nil
 	}
-	bytes, err := ioutil.ReadFile(strategiesFile)
+	bytes, err := ioutil.ReadFile(strategiesFile) /* nolint #nosec , this comes from an admin, not user */
```

Comment:
why does this require linter directives?

Reply:
The theory behind it is that `strategiesFile` can be manipulated by an end user, so that a file like `/etc/shadow` could be read. I tracked all instances of this case (G304, IIRC) and they all come from the CLI, meaning that they are set by an admin, and not by an end user.